### PR TITLE
🐛 fix: fix Add agent and Converse button not jump

### LINKED ISSUE #1782: [Bug] The button "Add agent and Converse" in the AgentDetail Sidebar doesn't jump.
### 💻 Operating System

Windows

### 📦 Environment

Official Preview

### 🌐 Browser

Chrome

### 🐛 Bug Description

After clicking **Add agent and Converse**, there is no message and no jump, but the agent is already added in the chat screen.

<br />

<br /><br />

### 🚦 Expected Behavior

After clicking **Add agent and Converse**, Jump directly to the corresponding Agent page to start chatting.

### 📷 Recurrence Steps

To https://chat-preview.lobehub.com/market?agent=gpts-big-fart-chat, then Click **Add agent and Converse**

### 📝 Additional Information

_No response_

### LINKED ISSUE #1784: [Bug] 使用“添加助手并会话”不会跳转到会话那里
### 💻 系统环境

Windows

### 📦 部署环境

Docker

### 🌐 浏览器

Chrome

### 🐛 问题描述
版本：0.142.6
添加助手并会话的功能，点击后没反应，不会回到会话界面。然后收到点击会话界面在会话界面是能看到这个助手添加成功。
image

### 🚦 期望结果

能正常跳转到会话界面进行会话

### 📷 复现步骤

发现-选择助手-添加助手并会话

### 📝 补充信息

_No response_

### PR DESCRIPTION
**Pull request by rogepi**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [x] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change

fix https://github.com/lobehub/lobe-chat/issues/1782
fix #1784

### CODE REVIEW

File: src/app/market/features/AgentDetailContent/AgentInfo/Header.tsx [hunk 1]
```diff
@@ -22,6 +24,21 @@ const Header = memo(() => {
   const { meta, createAt, author, homepage, config } = agentItem;
   const { avatar, title, description, tags, backgroundColor } = meta;
 
+  const handleAddAgentAndConverse = async () => {
+    if (!agentItem) return;
+
+    const session = await createSession({ config, meta });
+    message.success(t('addAgentSuccess'));
+    router.push(`/chat?session=${session}`);
```
Comment (arvinxx) [inline src/app/market/features/AgentDetailContent/AgentInfo/Header.tsx]: please use `SESSION_CHAT_URL` to adopt the switch of mobile and desktop:

https://github.com/lobehub/lobe-chat/blob/f0a3cbad488b03e1d96fdeb854c87e228266d89a/src/const/url.ts#L37-L38

```ts
  const isMobile = useIsMobile();

    router.push(SESSION_CHAT_URL(session,isMobile));
```

**Review by arvinxx — CHANGES_REQUESTED** [review]
Thanks for your contribution! just need a little change.😆

**Comment by rogepi** [comment]
Updated ~

**Review by arvinxx — APPROVED** [review]
LGTM, I'll merge it. Thanks for your contribution!

**Pull request merged**